controller/security_group.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_security_group" "AviatrixSecurityGroup" {
+  name        = "AviatrixSecurityGroup"
+  description = "Aviatrix - Controller Security Group"
+  vpc_id      = "${var.vpc}"
+
+  tags {
+    Name      = "AviatrixSecurityGroup"
+    Createdby = "Terraform+Aviatrix"
+  }
+}
+
+resource "aws_security_group_rule" "ingress_rule" {
+  type              = "ingress"
+  from_port         = 443
+  to_port           = 443
+  protocol          = "tcp"
+  cidr_blocks       = ["${var.incoming_ssl_cidr}"]
```

Comment:
Sane defaults, but allows the use of the module by tightening if the variable is set.  I'm open to changing the name, as it was first one that came to mind.

Reply:
I understand where this one is coming from, it's actually AWS Best practices to do this. 
However, if we do this by default, it might cause more support calls. The reason is that every Gateway tries to communicate back to controller to report "i'm alive". However, if we narrow down the allowed CIDR_Block, that Gateway will by default report as down and won't come up until you manually update the allowed CIDR_Block, therefore generating more support calls than necessary.  I do recommend tying down the CIDR_Block but on a customized basis.

Reply:
The default for the incoming_ssl_cidr is to leave things wide open, but it allows a user to limit this down.  So, the default behavior is to NOT lock things down (see the default value for incoming_ssl_cidr).

Reply:
Oh, The power of the defaults. Sounds good, will merge.